src/main/java/com/michelin/ns4kafka/services/TopicService.java

```diff
@@ -318,6 +319,18 @@ public Map<TopicPartition, Long> deleteRecords(Topic topic, Map<TopicPartition,
         }
     }
 
+    /**
+     * Check if all topic tags respect confluent format (starts with letter followed by alphanumerical or underscore).
+     *
+     * @param topic     The topic which contains tags
+     * @return true if yes, false otherwise
+     */
+    public Boolean isTagsFormatValid(Topic topic) {
+        return (topic.getSpec().getTags()
```

Comment:
There's an extra pair of parentheses that is not necessary (I'm suprised neither Sonar or Checkstyle reject it)

Reply:
Sure, I removed them